docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -452,51 +491,58 @@ main(){
     IDENTITIES_DIR="$PROFILE_DIR/identities"
 
     if (( DO_ZITI_LOGIN )); then
-        miniziti_login
+        minizitiLogin
         exit 0
     fi
+
     if (( SHOW_ADMIN_CREDS )); then
         showAdminCreds
         exit 0
     fi
 
-    if (( RUN_ZITI_CLI )); then
-        ziti_wrapper  "${ziti_cli_args[@]}"
+    if (( OPEN_ZITI_CONSOLE )); then
+        minizitiConsole
         exit 0
     fi
 
+    if (( RUN_ZITI_CLI )); then
+        zitiWrapper  "${ziti_cli_args[@]}"
+        exit 0
+    fi
 
-    (( DELETE_MINIZITI )) && {
+    if (( DELETE_MINIZITI )); then
         banner "$MINIKUBE_PROFILE"
-        deleteMiniziti 10
 
-        if (( MINIZITI_MODIFY_HOSTS )) && grep -q "$MINIZITI_PROFILE_MARKER" "$HOSTS_FILE"; then
-            checkSudoRequired
-            cleanHosts
+        trap EXIT
+        if ingress_zone="$(getIngressZone)"; then
```

Comment:
If the minikube node no longer exists this will fail to get the ingress zone and will still continue the miniziti delete operation because the errexit from the subshell is ignored in this case, an unfortunate quality of BASH error handling.

Alternatives include changing the function's `exit 1` to `return 1` and adding an `else; exit 1` to this `if` statement.

Reply:
Correct, also in case of partial 'start' failures so we can clean up.